pleskdistup/actions/extensions.py

```diff
@@ -32,3 +37,88 @@ def estimate_post_time(self) -> int:
 
     def estimate_revert_time(self) -> int:
         return 20
+
+
+# We should do rebundling of ruby applications after the conversion
+# because some of our libraries will be missing.
+# The prime example of missing library - libmysqlclient.so.18 required by mysql2 gem
+class RebundleRubyApplications(action.ActiveAction):
+    PLESK_APACHE_CONFIGS_PATH = "/etc/httpd/conf/plesk.conf.d/vhosts/"
+
+    def __init__(self):
+        self.name = "rebundling ruby applications"
+        self.description = "rebundling ruby applications"
+
+    def _is_ruby_domain(self, domain_path) -> bool:
+        return os.path.exists(os.path.join(domain_path, ".rbenv"))
+
+    def _is_required(self) -> bool:
+        if not os.path.exists("/var/lib/rbenv/versions/"):
+            return False
+
+        return any(self._is_ruby_domain(domain) for domain in os.scandir("/var/www/vhosts"))
```

Comment:
`/var/www/vhosts` also contains non-domain subdirectories like `chroot` or `system`.

Reply:
Yep, but we don't expect to find .rbenv directory in it. However we could simply exclude this two directories